src/main/java/de/tum/in/www1/artemis/service/LearningGoalService.java

```diff
@@ -278,20 +299,36 @@ public CourseLearningGoalProgress calculateLearningGoalCourseProgress(LearningGo
         // The progress will be calculated from a subset of the connected lecture units (currently only from released exerciseUnits)
         List<ExerciseUnit> exerciseUnitsUsableForProgressCalculation = learningGoal.getLectureUnits().parallelStream().filter(LectureUnit::isVisibleToStudents)
                 .filter(lectureUnit -> lectureUnit instanceof ExerciseUnit).map(lectureUnit -> (ExerciseUnit) lectureUnit).collect(Collectors.toList());
-        Set<CourseLearningGoalProgress.CourseLectureUnitProgress> progressInLectureUnits = this.calculateExerciseUnitsProgressForCourse(exerciseUnitsUsableForProgressCalculation,
+        Set<CourseLearningGoalProgress.CourseLectureUnitProgress> progressInExerciseUnits = this.calculateExerciseUnitsProgressForCourse(exerciseUnitsUsableForProgressCalculation,
                 useParticipantScoreTable);
 
+        // Temporary workaround to get number of possible participants in course
+        final Double numberOfStudents = userRepository.countUserInGroup(courseRepository.findByIdElseThrow(learningGoal.getCourse().getId()).getStudentGroupName()).doubleValue();
```

Comment:
I'd leave it as an int and instead multiply down below by 100.0 before dividing

Reply:
`countUserInGroup` returns a long, not int